capstone-frontend/src/components/SearchBookModal.jsx

```diff
@@ -222,11 +220,13 @@ export class SearchBookModal extends Component {
                 )
               }
 
-              <div className="text-center">
-                <Button className="text-center" variant="primary" onClick={async () => await this.handleSubmit()} >
-                  Add
+              {this.props.type !== "club"
+                && <div className="text-center">
+                  <Button className="text-center" variant="primary" onClick={async () => await this.handleSubmit()} >
```

Comment:
Same here: pretty sure the `async` and `await` are unnecessary here

Reply:
This part is only used for Mauro's BookLists so I'll check with him before changing anything, but I think you're right.

Reply:
Spoke with Mauro, we changed and it works fine!